assertions/scanner.go

```diff
@@ -618,9 +618,20 @@ func (m *Manager) keepTryingAssertionConfirmation(ctx context.Context, assertion
 		case <-ctx.Done():
 			return
 		case <-ticker.C:
-			// Assertions that are specified as the claim id in a challenge
-			// cannot be confirmed by time, so we exit early.
-			if m.challengeReader.IsClaimedByChallenge(assertionHash) {
+			parentAssertion, err := retry.UntilSucceeds(ctx, func() (protocol.Assertion, error) {
+				return m.chain.GetAssertion(ctx, protocol.AssertionHash{Hash: creationInfo.ParentAssertionHash})
```

Comment:
Given we are in the for loop, no need for the retry call as we can just log error and continue